### PR TITLE
add Maintainers

### PR DESCRIPTION
**Pull request by xmulligan**

### CODE REVIEW

File: roles/Maintainers.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
 # Maintainers
 
 The Maintainers roles and responsibilities are listed in [CONTRIBUTOR-ROLES.md](/CONTRIBUTOR-ROLES.md#maintainers). The following are the current maintainers:
+
+## Cilium
```
Comment (aanm) [inline roles/Maintainers.md:5]: What about cilium/proxy?

Comment (xmulligan) [reply roles/Maintainers.md:5]: I've added it below. Are there any other ones you think we should add?

**Review by joestringer — APPROVED** [review]

File: roles/Maintainers.md (hunk 1 above)
Comment (aanm) [reply roles/Maintainers.md:5]: Yes, I thought I saw it before, cilium/cilium-cli

Comment (xmulligan) [reply roles/Maintainers.md:5]: I've added Tobias there. I think that is it?

Comment (aanm) [reply roles/Maintainers.md:5]: Michi is also part of it https://github.com/orgs/cilium/teams/cilium-cli-maintainers

File: roles/Maintainers.md [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+
+* [Joe Stringer](https://github.com/joestringer)
+* [André Martins](https://github.com/aanm)
+
+## cilium-cli
+
+* [Tobias Klauser](https://github.com/tklauser)
```
Comment (xmulligan) [inline roles/Maintainers.md:12]:
```suggestion
* [Tobias Klauser](https://github.com/tklauser)
* [[Michi Mutsuzaki](https://github.com/michi-covalent)
```

File: roles/Maintainers.md (hunk 1 above)
Comment (xmulligan) [reply roles/Maintainers.md:5]: added

Comment (aanm) [reply roles/Maintainers.md:5]: Not easily no :(

**Review by aanm — APPROVED** [review]

**Pull request merged**